src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js

```diff
@@ -7,18 +7,19 @@ import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
 import { formatPercentage } from 'common/format';
 import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellLink from "../../../../common/SpellLink";
 
 class CorruptionUptime extends Module {
   static dependencies = {
     enemies: Enemies,
   };
 
-  suggestions(when){
+  suggestions(when) {
     const corruptionUptime = this.enemies.getBuffUptime(SPELLS.CORRUPTION_DEBUFF.id) / this.owner.fightDuration;
-    if(this.owner.selectedCombatant.hasBuff(SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id)) {
+    if (this.owner.selectedCombatant.hasBuff(SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id)) {
       when(corruptionUptime).isLessThan(0.95)
         .addSuggestion((suggest, actual, recommended) => {
-          return suggest('Your Corruption uptime can be improved. Try to pay more attention to your Corruption on the boss, especially since you have the T20 2-piece set bonus.')
+          return suggest(<span>Your Corruption uptime can be improved. Try to pay more attention to your Corruption on the boss, which is especially important with the <SpellLink id={SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id} children='T20 2-piece set bonus'/>.</span>)
```

Comment:
Children should always be specified like so:

`<SpellLink id={SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id}>T20 2-piece set bonus</SpellLink>`

They're kind of a special property that you shouldn't use directly.